docs/pages/database-access/guides/gui-clients.mdx

```diff
@@ -192,3 +192,36 @@ keep the Password field empty:
 ![DataGrip connection options](../../../img/database-access/guides/sqlserver/datagrip-connection@2x.png)
 
 Click OK to connect.
+
+## Redis Insight
+
+<Admonition type="note">
+  Teleport's Redis Insight integration only supports Redis standalone instances.
+</Admonition>
+
+After opening Redis Insight click `ADD REDIS DATABASE`.
+
+![Redis Insight Startup Screen](../../../img/database-access/guides/redis/redisinsight-startup.png)
+
+Click `Add Database Manually`. Use `127.0.0.1` as the `Host` and port printed by `tsh proxy db`.
+
+<Admonition type="note">
+  Remember to log in to your Redis instance with a Redis user first by using:
+
+  `tsh db login --db-user=alice redis-db-name`.
+</Admonition>
+
+Provide your Redis username as `Username` and password as `Password`.
+
+![Redis Insight Configuration](../../../img/database-access/guides/redis/redisinsight-add-config.png)
+
+Next, check the `Use TLS` and `Verify TLS Certificates` boxes and copy the CA certificate returned by `tsh proxy db`.
```

Comment:
We should include the instructions to copy the certificate when we first tell the reader to run `tsh proxy db`. Otherwise, they may have cleared the terminal at this point. (Unless the proxy runs in the foreground? I haven't run this command myself.)

Reply:
That's helpful, thanks. To make this clearer, I think we should replace line 210 with something like the following (removing the code fence escapes):

```markdown
Run the following command to start the Database Access proxy for your
Redis instance.

\`\`\`code
$ tsh proxy db redis 
Started DB proxy on 127.0.0.1:51823
Use following credentials to connect to the redis proxy: 
  ca_file=path/to/example.com.pem 
  cert_file=path/to/example.com/redis-x509.pem
  key_file=path/to/example.com/bob
\`\`\`


In Redis Insight, click `Add Database Manually`. Use `127.0.0.1` as the
`Host`. Copy the port printed by `tsh proxy db` and use it for the 
`Port` field.
```